Hyper/Scene.cs

```diff
@@ -12,13 +13,16 @@
 using Hyper.UserInput;
 using OpenTK.Mathematics;
 using OpenTK.Windowing.GraphicsLibraryFramework;
+using System.Collections.Concurrent;
 
 
 namespace Hyper;
 
 internal class Scene : IInputSubscriber
 {
-    private readonly List<Chunk> _chunks;
+    public readonly ConcurrentDictionary<Guid, Chunk> _existingChunks;
```

Comment:
ExistingChunks

Reply:
Also mayber remove _existingChunks and add a method to get them from chunkWorker

Reply:
_existingChunks is useful there is useful when Mine and Build methods are called